xonsh/jobs.py

```diff
@@ -123,21 +130,21 @@ def _set_pgrp(info):
     def _shell_tty():
```

Comment:
@scopatz On `_shell_tty()` thing, I'll do self review on it again later. but basically it was not needed since we starting record what happened to the owner of terminal, and an "am I own the terminal" is not needed anymore.